### PR TITLE
Import svg to services in config

### LINKED ISSUE #1629: Use @bbc/psammead-brand@1.0.0
Blocked on https://github.com/bbc/psammead/issues/199

**Is your feature request related to a problem? Please describe.**
Following https://github.com/bbc/psammead/issues/199 psammead-brand now takes an SVG object as a prop, to dynamically set the svg per service.

**Describe the solution you'd like**
Importing each services SVG object from @bbc/psammead-assets@0.1.8 into their service config, reading from the `ServiceContext` pass the svg group into the brand using @bbc/psammead-brand@1.0.0

This is only required for the 6 initial simorgh service (2 article, 4 ws)

**Testing notes**
[Tester to complete]

Dev insight: Visual checks, unit snapshot tests, storybook :) 

**Additional context**

- [x] Initially labelled with ["Refinement needed"](https://github.com/bbc/simorgh/labels/Refinement%20Needed)

### PR DESCRIPTION
**Pull request by ghost**
Resolves #1629 

**Overall change:** Added service svg's to config to be used on the `Brand` container

**Code changes:**

- Updated `psammead-assets` to `0.1.8` and `psammead-brand` to `1.0.0`
- Added svg's to their respective service configs
- Added `svg` prop value to `Brand` to display the correct service svg from the `ServiceContext`

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval
- [x] I have followed [the merging checklist](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/MERGE_PROCESS.md) and this is ready to merge.

### CODE REVIEW

File: src/app/lib/config/services/index.jsx [hunk 1]
```diff
@@ -18,6 +18,15 @@ export default {
     imageCopyrightOffscreenText: 'Default image copyright text, ',
     service: 'default',
     themeColor: '#000',
+    svg: {
```
Comment (dr3) [inline src/app/lib/config/services/index.jsx]: Hmm, not sure of this, but also not sure what to put in its place

File: src/app/lib/config/services/news.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import { C_POSTBOX } from '@bbc/psammead-styles/colours';
+import { news as svg } from '@bbc/psammead-assets/svgs';
```
Comment (dr3) [inline src/app/lib/config/services/news.js]: Can you change these to `brandSVG` please

File: src/app/lib/config/services/persian.jsx [hunk 3]
```diff
@@ -95,6 +96,7 @@ const persian = {
       },
     },
   },
+  svg: { group: <g />, viewbox: { height: 24, width: 167.95 }, ratio: 6.9979 },
```
Comment (dr3) [inline src/app/lib/config/services/persian.jsx]: For now, Persian can just use news's SVG

**Review by dr3 — CHANGES_REQUESTED** [review]

File: src/app/lib/config/services/index.jsx (hunk 1 above)
Comment (ghost) [reply src/app/lib/config/services/index.jsx]: Mayb we can put an empty group `<g/>`

Comment (dr3) [reply src/app/lib/config/services/index.jsx]: I would rather make SVGs optional in psammead-brand and not include one

Comment (dr3) [reply src/app/lib/config/services/index.jsx]: I would almost be tempted to leave it out of the defaults, they really need to be redone and nothing uses them/falls back to them. 

@sareh @pjlee11 @jblaut what do you think?

Also

Ive raised https://github.com/bbc/psammead/issues/472 to catch an error I found if the svg isnt provided

Comment (pjlee11) [reply src/app/lib/config/services/index.jsx]: What is the SVG we are suggesting inside of the `<g>`? I think without UX input we probably shouldn't define a default SVG, happy to leave it out and open a new issue so we don't hold up this PR

File: src/app/lib/config/services/persian.jsx [hunk 4]
```diff
@@ -1,4 +1,5 @@
 import { C_POSTBOX } from '@bbc/psammead-styles/colours';
```
Comment (dr3) [inline src/app/lib/config/services/persian.jsx]: Howcome this file is now `jsx`?

File: src/app/lib/config/services/index.jsx [hunk 5]
```diff
@@ -2,7 +2,6 @@
   This file includes data for the Service Context
```
Comment (dr3) [inline src/app/lib/config/services/index.jsx]: This file is now `jsx` too?

File: src/app/containers/Brand/index.jsx [hunk 6]
```diff
@@ -1,11 +1,11 @@
...
-import { ServiceContextConsumer } from '../../contexts/ServiceContext';
+import { ServiceContext } from '../../contexts/ServiceContext';
 
-const BrandContainer = () => (
-  <ServiceContextConsumer>
-    {({ brandName }) => <Brand brandName={brandName} />}
-  </ServiceContextConsumer>
-);
+const BrandContainer = () => {
+  const { brandName, brandSVG } = useContext(ServiceContext);
```
Comment (dr3) [inline src/app/containers/Brand/index.jsx:6]: `containers/Brand/index.test.jsx` needs to have a context provider as the svg isn't showing in the snapshots

File: src/app/containers/Header/__snapshots__/index.test.jsx.snap [hunk 7]
```diff
@@ -1,17 +1,6 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`Header should render correctly 1`] = `
```
Comment (dr3) [inline src/app/containers/Header/__snapshots__/index.test.jsx.snap:3]: Sorry @sadick254, The header tests need context too, my bad I should have said

**Review by dr3 — APPROVED** [review]
lgtm

**Review by DenisHdz — APPROVED** [review]
👍

**Comment by jamesbrumpton** [comment]
LGTM. Happy for this to be merged.

**Review by pjlee11 — APPROVED** [review]
Nice work!

**Pull request merged**